lib/src/components/_stacks-tags.less

```diff
@@ -0,0 +1,119 @@
+//
+//  TAGS
+//
+//  TABLE OF CONTENTS
+//  • BASE STYLE
+//  • STYLE VARIATIONS
+//  • LAYOUTS & SIZES
+//
+//  ===========================================================================
+//  $   MIXIN
+//      Any tag we use will always modify the same items, so to save time,
+//      here's a mixin to state all that CSS.
+//  ---------------------------------------------------------------------------
+& { // create a single scope to load the config into, at least for now
+#stacks-internals #load-config();
+
+.tag-styles(@border: transparent, @background: transparent, @color: inherit) {
+    border-color: @border;
+    background-color: @background;
+    color: @color;
+
+    .s-tag--dismiss {
+        background-color: transparent;
+        color: fade(@color, 90%);
+    }
+
+    &:hover {
+        border-color: darken(@border, 5%);
+        background-color: darken(@background, 5%);
+        color: darken(@color, 5%);
+
+        .s-tag--dismiss {
+            background-color: fade(@color, 10%);
+            color: darken(fade(@color, 90%), 5%);
+
+            &:hover {
+                background-color: fade(@red-500, 90%);
+                color: @white;
+            }
+        }
+    }
+}
+
+
+//  ===========================================================================
+//  $   BASE STYLE
+//      The same style is applied to all tags. Do not modify the core style.
+//  ---------------------------------------------------------------------------
+.s-tag {
+    display: inline-flex;
+    position: relative;
+    align-content: center;
+    justify-content: center;
+    min-width: 0;
+    padding: 0.545em;
+    border-style: solid;
+    border-width: 1px;
+    border-radius: @tags-border-radius;
+    font-size: @fs-fine;
+    line-height: @lh-sm;
+    text-decoration: none;
+    white-space: nowrap;
+    cursor: pointer;
+
+    .tag-styles(@tags-border, @tags-background, @tags-color);
+
+    &:hover {
+        text-decoration: none;
+    }
+}
+
+//  $$  DISMISS ICON
+//      Style adjument to @Svg.ClearSm
+//  ---------------------------------------------------------------------------
+.s-tag--dismiss {
```

Comment:
Could we deviate from the current style here? The red circle could be problematic if themes choose a different tag color. Would something like this work?

![screen shot 2018-06-08 at 11 59 23 am](https://user-images.githubusercontent.com/1369864/41170773-6aaf6d1a-6b13-11e8-921b-d572080c625c.png)

Reply:
Working on a new tiny icon for this one.